splunk-otel-android-volley/src/main/java/com/splunk/rum/volley/VolleyTracingBuilder.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum.volley;
+
+import com.android.volley.toolbox.HttpResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanStatusExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.CapturedHttpHeaders;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpSpanStatusExtractor;
+
+/**
+ * A builder for {@link VolleyTracing}.
+ */
+public final class VolleyTracingBuilder {
+
+    private static final String INSTRUMENTATION_NAME = "io.opentelemetry.volley";
```

Comment:
Oh, I totally missed that one -- it probably should be something like
```suggestion
    private static final String INSTRUMENTATION_NAME = "com.splunk.android.volley";
```
cause it's not OTel instrumentation; it's our (Splunk) custom instrumentation.